Model/Config/Entity/Erasure.php

```diff
@@ -51,4 +52,18 @@ public function getAllowedStatesToErase(int|string|null $website = null): array
             $website
         ));
     }
+
+    /**
+     * @param int|string|null $website
+     *
+     * @return int
+     */
+    public function getDelay(int|string|null $website = null): int
+    {
+        return (int)$this->scopeConfig->getValue(
+            self::CONFIG_PATH_ERASURE_DELAY,
+            ScopeInterface::SCOPE_WEBSITE,
+            $website
+        );
+    }
```

Comment:
Sorry, last comment 😅 
Because this setting is global and not scopable: https://github.com/opengento/magento2-gdpr/blob/ac18010522dda7c2ea0091a0482f5e7e6df6407e/etc/adminhtml/system/erasure.xml#L30

I don't think it makes sense to allow to pass a scope code. We can stuck to `(int)$this->scopeConfig->getValue(self::CONFIG_PATH_ERASURE_DELAY);` unless you may think it would be interesting to make this config scopable?

Reply:
Hum, I always try to have website / store configuration, it's the main feature of magento ?

Reply:
It's seem to be difficult to get website id in 

```
    private function retrieveScheduledAt(): string
    {
        return $this->localeDate->gmtDate(
            DateTimeFormat::DATETIME_PHP_FORMAT,
            $this->erasureConfig->getDelay() * 60 + $this->localeDate->gmtTimestamp()
        );
    }
```    

Solution : Store website id in database ?
Or we stay with global value

Reply:
I remove website from the configuration